File: WebApp/WebApp/Models/TimeTable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class TimeTable
    {
        public int Id { get; set; }
        public RideType RideType { get; set; }
        public DayOfWeek Day { get; set; }
        public DateTime Departure { get; set; }
        public Line Line { get; set; }
    }
}

File: Frontend/src/app/api/models/timetable.ts
/* tslint:disable */
import { DayOfWeek } from './day-of-week';
import { Line } from './line';
export interface Timetable {
  dayOfWeek?: DayOfWeek;
  departures?: null | Array<number>;
  id?: number;
  line?: null | Line;
}
